tsconfig.json

```diff
@@ -13,10 +13,11 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
     "noErrorTruncation": true,
-    "lib": ["es6", "esnext.asynciterable"]
+    "lib": ["dom", "es6", "esnext.asynciterable"]
```

Comment:
`lib` is already overridden in both the client and server side tsconfig files, maybe we don't need it here too?

Reply:
This `tsconfig` currently servers for ts files outside of src `scripts` and `config` so it is required. I added the `dom` because typescript requires all types in `@types` and `src/types/*.d.ts` and some types needs that even though we don't really use it. Another way would be to use a whitelist. I'd put more thought in it later, when we have e2e tests back, which probably would live in a `test` root level folder which would require additional tsconfig tweaking.